### PR TITLE
chore: update installation with docker

### PR DESCRIPTION
**Pull request by jarqvi**

### CODE REVIEW

File: .github/workflows/release-notes.py [hunk 1]
```diff
@@ -19,7 +19,7 @@
 Run a single node as follows:
 ```
 docker pull rqlite/rqlite
-docker run -p4001:4001 rqlite/rqlite
+docker run -d -p 4001:4001 -v data:/rqlite rqlite/rqlite
```
Comment (otoolep) [inline .github/workflows/release-notes.py:22]: Can you explain why we should enable this flags?

Comment (jarqvi) [reply .github/workflows/release-notes.py:22]: Thus, when I ran this container, I saw logs that apparently information such as snapshots are stored in the /rqlite path, so I preferred to store this data in a volume.
And I used the -d flag to run this container in the background.
240517_19h15m25s_screenshot

**Comment by otoolep** [comment]
The goal of the Docker command is to show the simplest possible way to run rqlite.

File: .github/workflows/release-notes.py (hunk 1 above)
Comment (jarqvi) [reply .github/workflows/release-notes.py:22]: OK, so at least it's good to add this to the documentation that important data is in the /rqlite path.

**Pull request closed without merging**